tests/spread/ci/task.yaml

```diff
@@ -0,0 +1,53 @@
+summary: Build an LXD spread test image with TPM software
+manual: true
+
+execute: |
+    INSTANCE_NAME=spread-test
+    IMAGE_NAME=lxd-spread-core22-img
+
+    # be in project folder
+    cd $PROJECT_PATH
+
+    # install prerequisites
+    snap install google-cloud-sdk --classic
+    snap install lxd --channel=latest/stable
+    lxd init --auto
+
+    # authenticate with gcloud
+    gcloud auth activate-service-account --key-file="./sa.json"
+
+    # launch the new lxc instance from an ubuntu 20.04 base
+    lxc launch ubuntu:20.04 $INSTANCE_NAME
+
+    # wait a few seconds before proceeding, otherwise we are going to have issues
+    # with the container (systemd) not being ready yet, and then the rest of
+    # the commands will fail
+    sleep 5s
```

Comment:
I wonder if we could simply do: `retry -n20 lxc exec $INSTANCE_NAME -- bash -c 'systemctl is-system-running '` here?

Reply:
Worked with some minor modifcations! thanks for the suggestions.